docs/index.md

```diff
@@ -413,3 +412,5 @@ ReactDOM.render(<App/>, mountNode);
 ## Documentation, Contributing, and Source
 
 Congratulations! You’ve created your first chart with Victory. For more information about Victory and its components, check out the docs - see [VictoryChart](http://formidable.com/open-source/victory/docs/victory-chart) to get started. Interested in helping out or seeing what's happening under the hood? Victory is maintained at [github.com/FormidableLabs/victory](https://github.com/FormidableLabs/victory), and you can [start contributing here](https://github.com/FormidableLabs/victory/#contributing). Happy charting.
+
+[our guides]: http://formidable.com/open-source/victory/guides
```

Comment:
We’re on https:// ! There is a redirect now, but not sure if it’s important here.